lib/hexpm/accounts/audit_log.ex

```diff
@@ -25,6 +25,19 @@ defmodule Hexpm.Accounts.AuditLog do
     }
   end
 
+  def build(%User{id: user_id}, user_agent, action, organization)
+      when action == "organization.create" do
```

Comment:
Pattern match directly instead:

```elixir
def build(%User{id: user_id}, user_agent, "organization.create", organization) do
```

Reply:
Okay! Resolved in 23e2c4ffe2cf20814f74e618dad040539130ae50